### ISSUE TITLE
🐛 Bug: (Self-hosted) Webhook between Docker containers on the same private network fails with unhelpful error message.

### ISSUE THREAD

**Issue by Banbury** [nocodb/nocodb#7644]
### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

I'm trying to call a webhook on n8n from Nocodb. I can call the webhook from a different dockerized app.

- Create docker containers n8n and Nocodb
- create a docker network and add both containers. 
- Nocodb should now be able to connect to n8n using the name of the container
- Create a workflow in n8n with a webhook trigger and activate it
- Define the corresponding webhook call in Nocodb (in my case the URL looks like this http://n8n-n8n-1:5678/webhook/7d6a6c0e-0a51-46ff-a95f-d522c12ce55f)
- Click 'Test Webhook'

I get an error message:
`DNS lookup 172.18.0.2(family:4, host:n8n-n8n-1) is not allowed. Because, It is private IP address.`

After further searching, I found the solution of setting the environment variable NC_ALLOW_LOCAL_HOOKS=true.
But the error message is not very helpful.

### Desired Behavior

There should be a better error message describing the problem and solution.

### Project Details

Node: **v18.18.2**
Arch: **x64**
Platform: **linux**
Docker: **true**
RootDB: **mysql2**
PackageVersion: **0.204.0**

**Comment by emre1e** [comment]
just try NC_ALLOW_LOCAL_HOOKS: 'true' as env variable

**Comment by Banbury** [comment]
Yeah, I know now. But it took me a while to figure that out. A better error message would go a long way.
